### DISCUSSION TITLE
More page feedback options

### DISCUSSION THREAD

**Discussion by isle9** [GitbookIO/community#359] [Feature Requests]
Hey, I was wondering if there are any plans for implementing more / better feedback options for pages? Currently, you can only collect page ratings which isn't really that useful if the users don't get a chance to elaborate on their ratings.

**Comment by rodrcastro** [comment]
Hey @isle9 👋 We sadly don't have plans to add more options on that front. However, [with our integrations platform](https://developer.gitbook.com/integrations/integrations), it should be possible to build something that allows you to gather more feedback. It might be a bit harder right now as the scope to inject code to a page is for internal-use only, [but I know that HotJar](https://help.hotjar.com/hc/en-us/articles/115011787508-What-is-a-Hotjar-Survey-) has an option to add surveys to a page - and it's an [integration we support currently :)](https://www.gitbook.com/integrations/hotjar)

Hopefully this is helpful!
